dev/NavigationView/NavigationView_rs1_themeresources.xaml

```diff
@@ -482,9 +483,6 @@
         </Setter>
     </Style>
 
-
-    <Thickness x:Key="NavigationViewItemBorderThickness">1</Thickness>
```

Comment:
Unrelated change?

Reply:
It was declared twice, I kept the other one at [L226](https://github.com/microsoft/microsoft-ui-xaml/blob/f89bbea0c9febcf0bcf7fa868876fb9a4aca6f58/dev/NavigationView/NavigationView_rs1_themeresources.xaml#L226).